<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng Ký - Task Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f0f0f0;
}
QLabel#title {
    font-size: 24px;
    font-weight: bold;
    color: #2c3e50;
}
QLineEdit {
    padding: 10px;
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    font-size: 14px;
}
QLineEdit:focus {
    border-color: #3498db;
}
QPushButton {
    padding: 10px;
    background-color: #27ae60;
    color: white;
    border: none;
    border-radius: 5px;
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #229954;
}
QPushButton:pressed {
    background-color: #1e8449;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>ĐĂNG KÝ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên đăng nhập:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>125</y>
      <width>340</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nhập tên đăng nhập</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mật khẩu:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>205</y>
      <width>340</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Nhập mật khẩu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_confirm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Xác nhận mật khẩu:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_confirm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>285</y>
      <width>340</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Nhập lại mật khẩu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_register">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Đăng Ký</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #95a5a6;
}
QPushButton:hover {
    background-color: #7f8c8d;
}
QPushButton:pressed {
    background-color: #5d6d7e;
}</string>
    </property>
    <property name="text">
     <string>Quay Lại</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
